7_react/react1/week1/vite-project/src/components/TodoListRenderer.jsx

```diff
@@ -0,0 +1,18 @@
+import ListingItems from "./ItemListDisplay.";
+
+function IteratingTodoItemsArray({ toDoItems }) {
```

Comment:
here is the same, change the name for component and your file, so it is matching and descriptive.